qualtran/serialization/data_types.py

```diff
@@ -86,3 +88,83 @@ def data_type_from_proto(serialized: SerializedDataTypes) -> QDType:
             "QAny, QInt, QIntOnesComp, QUInt, BoundedQUInt, "
             "QFixedPoint"
         )
+
+
+def int_or_sympy_from_qdt_proto(val: data_types_pb2.QDataType) -> Union[int, sympy.Expr]:
+    if val.HasField('qbit'):
+        return 1
```

Comment:
I think we should raise a value error if `val.HasField('qbit')` is true ? We've already checked the callsite and expect that it won't happen

Reply:
One could call this function from outside of data_type_to_qdt_proto no?

Reply:
Done.